mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_backup_plan_association.go

```diff
@@ -62,11 +62,6 @@ func DataSourceGoogleCloudBackupDRBackupPlanAssociations() *schema.Resource {
 				Computed:    true,
 				Description: "The ID of the project in which the resource belongs.",
 			},
-			"resource_type": {
```

Comment:
Just wondering, if this field is removed, what happens to customers who are already using it in their config?